test/y2storage/clients/partitions_proposal_test.rb

```diff
@@ -257,20 +255,18 @@
           end
         end
 
-        context "and the expert partitioner returns :next" do
-          let(:result) { :next }
+        context "and the storage client returns :cancel" do
+          let(:result) { :cancel }
 
-          before do
-            allow(expert_dialog).to receive(:device_graph).and_return(new_devicegraph)
+          it "returns a hash with :back for 'workflow_sequence' key" do
+            result = subject.ask_user(param)
+            expect(result).to be_a(Hash)
```

Comment:
NP: I think that line is not really necessary. The above expectation should be enough.

Reply:
Well, due to the nature of `#ask_user` it should return a hash, I would prefer to check it explicitly.